doc/sphinx/inter_non-bonded.rst

```diff
@@ -216,6 +216,40 @@ interaction, while :math:`\delta` varies how smoothly the potential goes to zero
 alchemical transformations, where a group of atoms can be slowly turned
 on/off during a simulation.
 
+.. _Weeks-Chandler-Anderson interaction:
+
+Weeks-Chandler-Anderson interaction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    Feature ``WCA`` required.
+
+
+The interface for the Weeks-Chandler-Anderson interactions is implemented in
+:class:`espressomd.interactions.WCAInteraction`. They
+are configured via the syntax::
+
+    system.non_bonded_inter[type1, type2].wca.set_params(**kwargs)
+
+This command defines a Weeks-Chandler-Anderson interaction between particles of the
+types ``type1`` and ``type2``. The potential is defined by
+
+.. math::
+
+   \label{eq:wca}
+     V_\mathrm{WCA}(r) =
+       \begin{cases}
+         4 \epsilon \left[ \left(\frac{\sigma}{r}\right)^{12}
```

Comment:
It may be handy to allow an ```offset``` similar to the offset in the Lennard-Jones interaction.

Reply:
My point was to rather keep it simple, as there is already a Lennard-Jones that has all the bells and whistles.

Reply:
It's open for extension, so I'm okay